athena-google-bigquery/src/test/java/com/amazonaws/athena/connectors/google/bigquery/BigQueryFederationExpressionParserTest.java

```diff
@@ -265,7 +265,7 @@ public void testComplexExpressions_Simple_With_Date()
                 ImmutableList.of(colOne, dateConstantExpression));
 
         String s = bigQueryExpressionParser.parseFunctionCallExpression((FunctionCallExpression) gtFunctionCall);
-        assertEquals("(colOne > '2020-01-05')", bigQueryExpressionParser.parseFunctionCallExpression((FunctionCallExpression) gtFunctionCall));
+        assertEquals("(colOne > '2020-01-04')", bigQueryExpressionParser.parseFunctionCallExpression((FunctionCallExpression) gtFunctionCall));
```

Comment:
What is the reason for this? Pulled the PR locally and this caused the test to fail. I assume its a mistake?

Reply:
It was faiing in my local, so i changed. but let me revert this in this case.